base.docker

```diff
@@ -17,8 +21,8 @@ RUN apt-get update && \
     apt-get update && \
     apt-get install -y --no-install-recommends yarn=${YARN_VERSION} && \
     rm -rf /var/lib/apt/lists/* && \
-    /usr/bin/python2.7 /usr/bin/pip install supervisor==3.3.3 && \
-    pip install --no-cache-dir uwsgi awscli awscli-cwlogs && \
+    /usr/bin/python2.7 /usr/bin/pip install supervisor==${SUPERVISOR_VERSION} && \
+    /usr/local/bin/pip3 install --no-cache-dir uwsgi==${UWSGI_VERSION} awscli==${AWSCLI_VERSION} awscli-cwlogs==${AWSCLI_CWLOGS_VERSION} && \
```

Comment:
Is it used mostly when we deploy code, or in development environments, too?

Reply:
This is primarily for deployed applications, although you can use the docker containers locally as well.

Reply:
Ok, cheers ☺️